namelists/echam/6.3.04p1/cmip6/namelist.echam

```diff
@@ -0,0 +1,139 @@
+&parctl
+	nproca=24
+	nprocb=48
+/
+&runctl
+	NPROMA=72
+	OUT_FILETYPE=2
+	OUT_ZTYPE=0
+	RERUN_FILETYPE=4
+	LTIMER=.false.
+	LMIDATM=.true.
+	LFRACTIONAL_MASK=.false.
+	!LHD=.true.
+	LCOUPLE=.true.
+	LCOUPLE_CO2=.true.
+	!LIPCC=.true.
+	LAMIP=.false.
+	PUTOCEAN=1,'days','last',0
+	GETOCEAN=1,'days','last',0
+	PUTDATA=24,'hours','last',0
+	default_output=.true.
+	TRIGFILES=12,'months','exact',0
+	LDEBUGEV=.false.
+	!lmeltpond=.false.
+	out_expname=From2017in2019in2017Climatessp3703
+	dt_stop=2019,10,01
+	dt_resume=2019,07,01
+	lresume=.true.
+	out_datapath='/work/ba1138/a270148/esm-experiments/AWICM/From2017in2019in2017Climatessp3703/work//'
+	delta_time=200
+	putrerun=3,'months','first',0
+/
+&submdiagctl
+	vphysc_lpost = .false.
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'spim'
+	source = 'sp'
+	variables = 'st:mean', 'svo:mean', 'lsp:mean', 'sd:mean'
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'glim'
+	source = 'gl'
+	variables = 'xl:mean', 'xi:mean'
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'aclcim'
+	source = 'g3b'
+	variables = 'aclc:mean'
+/
+&set_stream_element
+	stream = 'aclcim'
+	name = 'aclc'
+	code = 162
+	lpost = 1
+/
+&set_stream
+	stream = 'jsbach'
+	lpost = 0
+/
+&set_stream
+	stream = 'land'
+	lpost = 0
+/
+&set_stream
+	stream = 'veg'
+	lpost = 0
+/
+&set_stream
+	stream = 'yasso'
+	lpost = 0
+/
+&mvstreamctl
+	interval = 1, 'months', 'last', 0
+	target = 'g3bim'
+	source = 'g3b'
+	variables = 't2min:min', 't2max:max', 'topmax:mean'
+/
+&mvstreamctl
+	interval = 1, 'days', 'last', 0
+	target = 'glday'
+	source = 'gl'
+	variables = 'q:mean'
+/
+&mvstreamctl
+	target = 'g3bday'
+	interval = 1, 'days', 'last', 0
+	source = 'g3b'
+	meannam = 'tslm1', 'tsi'
+/
+&mvstreamctl
+	interval = 1, 'days', 'last', 0
+	target = 'g3bid'
+	source = 'g3b'
+	variables = 'u10:mean', 'v10:mean', 'temp2:mean', 'relhum:mean',
+	'dew2:mean', 'ws:mean', 'sn:mean', 'wimax:max', 't2min:min', 't2max:max'
+/
+&mvstreamctl
+	interval = 1, 'days', 'last', 0
+	target = 'jsbid'
+	source = 'jsbach'
+	variables = 'layer_moisture:mean'
+/
+&mvstreamctl
+	interval = 6, 'hours', 'last', 0
+	target = 'sp6h'
+	source = 'sp'
+	variables = 'st:mean', 'svo:mean', 'lsp:mean', 'sd:mean'
+/
+&mvstreamctl
+	interval = 1, 'hours', 'last', 0
+	target = 'g3b1hi'
+	source = 'g3b'
+	variables = 'u10:inst', 'v10:inst', 'wimax:max'
+/
```

Comment:
We have talked about this, but I at some point really would be happy to separate the model physical settings and the output choice settings, at least on the tools side as far as possible. Miguel suggested to merge namelists together with add and remove functionally at some point, I would strongly advocate to follow that way of thinking. But it will be tricky...

Determine model (I/)O (I guess O) is something I really at some point want to get a decent handle on.

Reply:
I think the tricky part would be to get an agreement on what we should provide as defaults selections for the output, and again defaults for the "physical" parts of the default echam and jsbach namelists. In technical terms, we have already a function that does something very similar, and in the end is a matter of extending it a bit to append some text. We can open a mini project for that.

Reply:
I have added a project for this, and therefore I'll mark this by resolved.